element/src/commonMain/kotlin/RootElement.kt

```diff
@@ -1,16 +1,67 @@
 package com.juul.krayon.element
 
+import com.juul.krayon.kanvas.IsPointInPath
 import com.juul.krayon.kanvas.Kanvas
+import com.juul.krayon.kanvas.Transform
+import com.juul.krayon.kanvas.Transform.Translate
 
 public class RootElement : Element() {
 
     override val tag: String get() = "root"
 
+    public var onClickFallback: (() -> Unit)? by attributes.withDefault { null }
+
     override fun <PATH> draw(canvas: Kanvas<PATH>) {
         children.forEach { it.draw(canvas) }
     }
 
+    /** Returns true if an element was found and clicked on. Returns false if no element matched. */
+    public fun onClick(isPointInPath: IsPointInPath, x: Float, y: Float): Boolean {
+        val interactable = visibilityOrderedDescendants()
```

Comment:
nit: Would be less confusing (at least to me) to name this variable `element` (or `child`)?
I kept reading the code assuming `interactable` was of type `Interactable<T>`.

```suggestion
        val element = visibilityOrderedDescendants()
```

Reply:
It is both an interactable and an element. Unfortunately we don't have union types yet. To hijack a syntax allowed elsewhere, this value is roughly `val interactable: T where T: Element, T: Interactable<T>` or `val interactable: Element & Interactable<out Element>`. Neither of which actually works here.